gpu: nvidia: ip: respect acc_mode for sum post-op (#2479)
oneapi-src/oneDNN: src/gpu/nvidia/README.md

The inner product primitives is an implementation of matrix multiplication plus
bias activation. There are two implementation of inner product in cuDNN backend.

> With `sum` post-op, the accumulation mode attribute affects behavior as

Reviewer: ```suggestion
With `sum` post-op, the accumulation mode attribute affects the behavior as
```

> With `sum` post-op, the accumulation mode attribute affects behaviour as

Reviewer 2: ```suggestion
With `sum` post-op, the accumulation mode attribute affects behavior as
```
American English is preferred

follows:

> - `relaxed`: Uses GEMM’s beta parameter for a fused, optimized sum post-op but

Reviewer: ```suggestion
- `relaxed`: Uses GEMM’s beta parameter for a fused and optimized `sum` post-op but
```

> - `relaxed`: Uses GEMM’s beta parameter for a fused, optimised sum post-op but

Reviewer 2: ```suggestion
- `relaxed`: Uses GEMM’s beta parameter for a fused, optimized sum post-op but
```
American English

Author: Done

  may reduce output precision for large `f16` inputs.

> - `strict` (default): Converts GEMM output to `f32`, performs sum as a separate

Reviewer: ```suggestion
- `strict` (default): Converts GEMM output to `f32`, performs `sum` post-op as a separate
```
...Perform sum post-op?

>   operation, then converts it back to the original type. This is more precise

Reviewer: ```suggestion
  operation, then converts it back to the original type. This attribute provides better output precision
```

>   but less performant.

Reviewer: ```suggestion
  but reduced performance.
```
